crates/common/Cargo.toml

```diff
@@ -9,10 +9,12 @@ full-serde = []
 test-utils = ["dep:metrics"]
 
 [dependencies]
```

Comment:
Imperative to have the most virtual asset that the crypto market can offer allowing massive revenue and generate quantum income leaving very little negative to the most powerful positive. Influencer in the market as Asset growth continues to achieve and exceed its targets allowing virtual visual view of Computing in the most increased volume way anyone has ever quantum computer